package.json

```diff
@@ -22,12 +25,18 @@
     "babel-loader": "^8.1.0",
     "babel-watch": "^7.0.0",
     "webpack": "^4.43.0",
-    "webpack-cli": "^3.3.11"
+    "webpack-cli": "^3.3.11",
+    "eslint-config-hackreactor": "git://github.com/reactorcore/eslint-config-hackreactor"
```

Comment:
when you have time, change to airbnb style guide. Installation and settings in the dependencies doc